guide/popular-topics/errors.md

```diff
@@ -0,0 +1,155 @@
+
+<branch version="12.x">
+
+### MessageEmbed field names may not be empty.
+
+This error originates from attempting to call `MessageEmbed.addField()` without the first parameter, which is a title. If you would like the title to be empty for a reason, you should use a zero width space, which can be inputted as `\u200b`.
+
+### MessageEmbed field values may not be empty.
+
+This error, in conjunction to the previous error, is the result of calling `MessageEmbed.addField()` without the second parameter, the value. You can use a zero width space if you would like this empty.
+
+</branch>
+
+### The messages must be an Array, Collection, or number.
+
+This error originates from an invalid call to `bulkDelete()`, make sure you are inputting a valid Array or Collection of messages, or a valid number.
+
+### Members didnt't arrive in time.
+
+Another common error, this error originates from the client requesting members from the API through the websocket, and the member chunks not arriving in time and triggering the timeout. The most common cause to this error is a bad connection, however, it can also be caused by a very large amount of members being fetched, upwards of 50 thousand. To fix this, run the bot on a location with better internet, such as a VPS. If this does not work for you, you will have to manually change the member fetching timeout.
+
+### MaxListenersExceededWarning: Possible EventEmitter memory leak detected...
+
+This error is caused by spawning a large amount of event listeners, usually for the client. The most common cause of this is nesting your event listeners instead of separating them. This can also be caused by spawning a lot of Collectors, since Collectors themselves spawn event listeners. The way to fix this error is to make sure you do not nest your listeners or spawn a lot of Collectors, it is **not** to use `emitter.setMaxListeners()` as the error suggests.
+
+### Cannot send messages to this user.
+
+This error is thrown when the bot attempts to send a DM message to a user and it is unable to do so. This is caused by a variety of reasons:
+- The bot and the user do not share a guild (oftentimes people accidentally kick or ban a user and then try to dm them).
+- The user has blocked the bot.
+- The user has disabled dms in the privacy settings.
+
+In the case of the last two reasons, the error is not preventable, as the Discord API does not provide a way to check if you can send a user a dm until you attempt to send one. The best way to handle this error is to add a `.catch()` where you attempt to dm the user, and either ignore the rejected promise, or do what you want because of it.
+
+## Common miscellaneous errors
+
+### code ENOENT... syscall spawn git.
+
+This is an error commonly thrown by npm due to git not being installed on your machine. You must install git:
```

Comment:
*Technically* this error is emitted by system -> node.js -> npm/yarn. Maybe reword this a bit?

Reply:
`This is an error commonly thrown by Node in response to npm not being able to find git`...?

Reply:
``This error is commonly thrown by your system due to it not being able to find `git`. You need to install `git` or update your path if `git` is already installed. Here are the download links for it:
- Links 
- Here
```
Or something that doesn't sound bad 😄